src/sugar/balances.ts

```diff
@@ -41,16 +41,19 @@ async function getBalances(
   options: GetBalancesOptions = {},
 ): Promise<Balance[]> {
   // 1. Get XRP Balance
-  const xrpRequest: AccountInfoRequest = {
-    command: 'account_info',
-    account,
-    ledger_index: options.ledger_index ?? 'validated',
-    ledger_hash: options.ledger_hash,
+  const xrpBalance: Balance[] = []
+  if (!options.peer) {
+    const xrpRequest: AccountInfoRequest = {
+      command: 'account_info',
+      account,
+      ledger_index: options.ledger_index ?? 'validated',
+      ledger_hash: options.ledger_hash,
+    }
+    const balance = await this.request(xrpRequest).then(
+      (response) => response.result.account_data.Balance,
+    )
+    xrpBalance.push({ currency: 'XRP', value: dropsToXrp(balance) })
```

Comment:
Why do we need an array for this. Is there a scenario where you can have two XRP balances?